src/static/clientConfig.ts

```diff
@@ -6,6 +6,7 @@
  */
 import type {RequestInit as NodeRequestInit} from 'node-fetch';
 import type {BaseUriParameters} from './helpers/types';
+import {fetch} from './helpers/environment';
```

Comment:
Are we using this fetch instance anywhere ? Might want to remove the import if it is unused.

Reply:
good call out, originally used it as a default argument but unused import has now been removed